.github/workflows/coverage.yml

```
@@ -0,0 +1,100 @@
+name: Coverage
+
+on:
+  pull_request_target:
+    paths:
+      - 'django/**/*.py'
+      - 'tests/**/*.py'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  diff-coverage:
+    name: Diff Coverage (Windows)
```

A couple other jobs do this. We can loosen it later, but let's only act on django/django for now.
```suggestion
    if: github.repository == 'django/django'
    name: Diff Coverage (Windows)
```